lib/src/util/react_util.dart

```diff
@@ -40,6 +40,8 @@ class UiPropsMapView extends MapView
   bool get $isClassGenerated =>
       throw UnimplementedError('@PropsMixin instances do not implement \$isClassGenerated');
 
+  PropsMetaCollection get $meta => throw UnimplementedError('@PropsMixin instances do not implement instance meta');
```

Comment:
Unrelated to this PR, but we should really deprecate the `UiPropsMapView` class... It's made obsolete by the new boilerplate for props map views, which are real `UiProps` classes